src/content/events/2024/hackathon-march-2024/usa-sanfrancisco.md

```diff
@@ -1,20 +1,18 @@
 ---
 title: Hackathon - March 2024 (San Francisco)
-subtitle: Local node of the nf-core hackathon at the Benchling Offices in San Francisco
+# subtitle: Local node of the nf-core hackathon at the Benchling Offices in San Francisco
+subtitle: Local node of the nf-core hackathon in San Francisco
 type: hackathon
 startDate: '2024-03-18'
 startTime: '09:00-06:00'
 endDate: '2024-03-20'
 endTime: '13:00-06:00'
-locationName: Benchling
-address: 680 Folsom St., 8th Floor San Francisco, CA 94107
-locationURL: https://maps.app.goo.gl/TJAGF7ypXFwwCcLG7
-locationLatLng: [37.78452954687462, -122.39858903879933]
+locationName: TBA
+# address: 680 Folsom St., 8th Floor San Francisco, CA 94107
+# locationURL: https://maps.app.goo.gl/TJAGF7ypXFwwCcLG7
+# locationLatLng: [37.78452954687462, -122.39858903879933]
 ---
 
-[Benchling](https://www.benchling.com/) have very kindly offered to host a local
-site for the nf-core hackathon in some space in their 10th floor office in downtown San Francisco(680 Folsom).
-
 More details coming soon. Please register via the main nf-core hackathon page,
```

Comment:
Please register via the main nf-core hackathon page, as we need to track attendee info to ensure access to the building.

Reply:
Something like that!